### PR TITLE
Reworks catnip

### PR DESCRIPTION
**Pull request by tf-4**
# WIP
Waiting on one or two other things like https://github.com/Skyrat-SS13/Skyrat-tg/pull/16081 and https://github.com/Skyrat-SS13/Skyrat-tg/pull/16078
## About The Pull Request

Catnip overhaul, felinid players rejoice.
Now catnip is a mutation of tea, along with Tea Astra as usual. It's been removed from the contraband section of the vendor.
It is no longer a subtype of pax. No idea why it was that way to start, and I added it in the first place.

When drinking frisky kitty, you now get a brain trauma! (Though only if you're a felinid).

This is a special, random roll disabled trauma that makes you hallucinate laser pointers around your person. You can also hallucinate mouse squeaks at varying distances.
Hallucinatory lasers will occasionally appear in any tile adjacent to you, and you have a 50% chance to pounce at it. The pounce can knock down people you dash into which will deal a small amount of brute damage (~10), but you won't grab them like gripper gloves. It does a small amount of stamina damage to you per pounce.

## How This Contributes To The Skyrat Roleplay Experience

The old catnip was weird and bad.
Adds a slightly more unique function to frisky kitty rather than it's boringness before

## Changelog
:cl:
add: Reworked frisky kitty to make you hallucinate laser dots around your person if you consume it as a felinid.
balance: Slightly lowered the export value of catnip. It's not that rare.
balance: Catnip is no longer a derivative of pax
fix: 
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
I'm not a fan of this personally as a felinid player. I was under the impression that species drinks were supposed to be a nice time for their intended drinkers. This feels like a bizarre reverse of drinks like beepsky smash which punish unintended imbibing. 

At best this annoys felinids with laser pointers (something that people already do to harass felinids). At worst this makes felinids really disruptive at the bar, jumping all over people uncontrollably and even doing damage to them. 

This feels bad for everyone involved and feels like a punishment for enjoying their species drink.

**Comment by Paxilmaniac** [comment]
You are drinking something made out of what is essentially just weed for your species, I don't know what you're expecting to be honest.

**Comment by YakumoChen** [comment]
I can't see "bar fight: the drink" being good at all for roleplay or the game

>something made out of what is essentially just weed for your species,

That's a bad and wrong precedent to make, most of the other species drinks have very strange ingredients. Hell, the lizard one is made of *slime jelly*.

**Comment by ghost** [comment]
> You are drinking something made out of what is essentially just weed for your species, I don't know what you're expecting to be honest.

Technically it depends how they consume it. Inhaling it is stimulating, while just eating is actually a depressant. (https://www.humanesociety.org/resources/crazy-catnip) _Info Source_
You COULD argue that a drink is a weird combination of both but the point DOES still apply, consuming it shouldn't make you a nuisance.

However
Making it possible to huff catnip to get the jittery effects
Would be kinda based..,,

File: modular_skyrat/modules/catnip/code/catnip_trauma.dm [hunk 1]
```diff
@@ -0,0 +1,77 @@
+/datum/brain_trauma/special/laser_pointer
+	name = "laser pointers"
+	desc = "Patient seems to be a feline."
+	scan_desc = "feline tendencies"
+	gain_text = "<span class='warning'>LIGHT.</span>"
+	lose_text = "<span class='notice'>Where'd that light go?</span>"
```
Comment (press-mark) [inline modular_skyrat/modules/catnip/code/catnip_trauma.dm:6]:
```suggestion
	span_warning("LIGHT.")
	span_notice("Where'd that light go?")
```

Comment (tf-4) [reply modular_skyrat/modules/catnip/code/catnip_trauma.dm:6]: Nothing I'm aware of uses them for gain and lose text, and I'm _somewhat_ sure there's a reason for it though I wouldn't be able to tell you what it was.

**Comment by tf-4** [comment]
Another time

**Pull request closed without merging**